courseraprogramming/commands/config.py

```diff
@@ -20,15 +20,77 @@
 You may install it from source, or via pip.
 """
 
+from courseraprogramming.commands import oauth2
```

Comment:
pep8 says that local library imports should be last: http://legacy.python.org/dev/peps/pep-0008/#imports